internal/server/openapi.yaml

```diff
@@ -452,6 +452,26 @@ paths:
           $ref: '#/components/responses/forbiddenError'
         '500':
           $ref: '#/components/responses/internalServerError'
+  /events/reports/{teamKey}:
```

Comment:
Now that there are two reports endpoints, we should probably have a `/reports` endpoint with query parameters. `/events/reports/{teamKey}` and `/events/{eventKey}/matches/{matchKey}/reports/{teamKey}` aren't talking about different resources.

Reply:
Makes sense to me 👍